libchisel/src/lib.rs

```diff
@@ -3,7 +3,7 @@ extern crate binaryen;
 extern crate parity_wasm;
 extern crate rustc_hex;
 
-pub use parity_wasm::elements::Module;
+use parity_wasm::elements::Module;
```

Comment:
Can do `pub use parity_wasm::elements::Module as WasmModule` if we also rename it in the rest of the codebase, which would have the side effect of making all the other code more readable.